src/ucm/event/event.c

```diff
@@ -469,13 +469,14 @@ int ucm_madvise(void *addr, size_t length, int advice)
     return event.madvise.result;
 }
 
-void ucm_library_init(const ucm_global_config_t *ucm_opts)
+void ucm_library_opts(const ucm_global_config_t *ucm_opts)
 {
-    UCS_INIT_ONCE(&ucm_library_init_once) {
-        if (ucm_opts != NULL) {
-            ucm_global_opts = *ucm_opts;
-        }
+    ucm_global_opts = *ucm_opts;
+}
 
+void ucm_library_init()
```

Comment:
> it seems that when we call ucm_library_opts from UCS, we always call ucm_library_init() after it.
> so maybe expose only ucm_library_opts() in ucp.h, and ucm_library_init could be an internal function?

ok, removed global declaration of ```ucm_library_init```, can't declare it as ```static``` because it is used in memhooks test